src/Widgets/Item/ItemDataTableWidget.php

```diff
@@ -3,8 +3,57 @@
 namespace Ceres\Widgets\Item;
 
 use Ceres\Widgets\Helper\BaseWidget;
+use Ceres\Widgets\Helper\Factories\Settings\ValueListFactory;
+use Ceres\Widgets\Helper\Factories\WidgetSettingsFactory;
+use Ceres\Widgets\Helper\WidgetCategories;
+use Ceres\Widgets\Helper\WidgetDataFactory;
+use Ceres\Widgets\Helper\WidgetTypes;
 
 class ItemDataTableWidget extends BaseWidget
 {
     protected $template = "Ceres::Widgets.Item.ItemDataTableWidget";
+
+    public function getData()
+    {
+        return WidgetDataFactory::make("Ceres::ItemDataTableWidget")
+            ->withLabel("Widget.itemDataTableLabel")
+            ->withPreviewImageUrl("/images/widgets/item-data-table.svg")
+            ->withType(WidgetTypes::ITEM)
+            ->withCategory(WidgetCategories::ITEM)
+            ->withPosition(600)
+            ->toArray();
+    }
+
+    public function getSettings()
+    {
+        /** @var WidgetSettingsFactory $settingsFactory */
+        $settingsFactory = pluginApp(WidgetSettingsFactory::class);
+
+        $settingsFactory->createCustomClass();
+
+        $settingsFactory->createSelect("itemInformation")
+            ->withName("Widget.")
+            ->withDefaultValue("item.id")
+            ->withList(true)
```

Comment:
withList nimmt 1 bis 2 Werte (min, max).

```suggestion
            ->withList(1)
```